.gitignore

```diff
@@ -9,3 +9,6 @@
 /tmp/
 footprint.sql
 footprint.:memory:.sql
+.idea/
```

Comment:
It would be better to add this to your `.gitignore_global` file rather than checking in intelliJ specific stuff here.  Presumably the same applies to the rbenv stuff.